### PR TITLE
Carry the argparse changes to osnode

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

**Review by jovial — APPROVED** [review]
Very nice - can't wait for upgrade on dev-director. Do we need to bump the VERSION file?

File: bin/osnode [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+    parser = argparse.ArgumentParser(description='query ironic nodes info by nova server name')
+    parser.add_argument('pattern', metavar='pattern',
+                        help='pattern to apply to server names, e.g. openhpc-')
+    parser.add_argument('query', metavar='query',
+                        help='json query to filter results by, e.g. .driver_info.ipmi_address')
+    parser.add_argument('--all-projects', dest='all',
+                        action='store_true',
+                        help='search all projects (requires admin credentials)'
+                        )
+    parser.set_defaults(all=True)
```
Comment (jovial) [inline bin/osnode:17]: Just thought, I meant to default this to false.

**Comment by brtkwr** [comment]
Done :) and version 0.2 is already installed on dev-director :)

File: bin/osnode (hunk 1 above)
Comment (brtkwr) [reply bin/osnode:17]: I've updated this on master

**Pull request merged**